content/blog/2024-04-04-tmix-03.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "This Month in Xilem, March 2024"
+authors = ["Daniel McNab"]
++++
+
+Since the last update, there have been three newly released projects from Linebender:
+
+- [![Velato's version on crates.io.](https://img.shields.io/crates/v/velato.svg)](https://crates.io/crates/velato)
+ [Velato][]: A parser and driver for Lottie animation files in applications using Vello
+- [![Vello SVG's version on crates.io.](https://img.shields.io/crates/v/vello_svg.svg)](https://crates.io/crates/vello_svg)
+  [Vello SVG][]: Allows displaying (some) SVG documents in Vello applications
+- [![Tracing Android Trace's version on crates.io.](https://img.shields.io/crates/v/tracing_android_trace.svg)](https://crates.io/crates/tracing_android_trace)
```

Comment:
I find the wording "version on crates.io" to be a little odd. If it's a new version, you can call it a vx.x.x release. If it's the first one, I think "published to crates.io" is clearer.

Reply:
The specific point of this text is to give an alt text to the badge.

So you'd prefer something like `v0.1.0 on crates.io.`?